tests/cxx/var_cleanup_attr.cc

```diff
@@ -0,0 +1,33 @@
+//===--- var_cleanup_attr.c - test input file for iwyu -------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// IWYU_ARGS: -I .
+
+// The cleanup attribute runs a function when the variable goes out of scope.
+// This test makes sure that we track that function.
+
+#include "tests/cxx/var_cleanup_attr-d1.h"
+
+void Cleanup() {
+  // IWYU: Freep is defined in...*which isn't directly #included.
```

Comment:
I think you want to check that it actually does recommend `var_cleanup_attr-i1.h` here, so;
```
// IWYU: Freep is...*var_cleanup_attr-i1.h
```
is the more typical way to cover that.